src/components/config/index.ts

```diff
@@ -161,9 +161,22 @@ export const createConfig = async (flags: any) => {
     open('https://hyperjumptech.github.io/monika-config-generator/')
   } else {
     const { path, type } = getPathAndTypeFromFlag(flags)
+
+    if (!existsSync(path)) {
+      log.error(`Couldn't found the ${path} file.`)
+      return
+    }
+
     const parsed = parseConfig(path, type)
     const file = flags.output || 'monika.json'
 
+    if (existsSync(file) && !flags.force) {
+      log.error(
+        `Cannot write configuration file to ${file} because file already exists.`
```

Comment:
I may need some feedback on this, but how about we use incremental filename?
https://stackoverflow.com/questions/41314177/how-to-increment-filename-if-file-exists

Let's say that monika.json is already exists, when there is no --force flag, it will check if there is any file named monika.json in the directory. 
- If it exists, create monika-1.json
- If it doesn't exists, proceed with monika.json
- If there is monika and monika-1.json, it will be monika-2.json
- If there is only monika-1.json, proceed with monika.json

But then again, it's OK if we stay with this

Reply:
+1 with Denny idea

Reply:
There's the -o option for this already. having number is ambiguous and hard to find.